ISISComputingGroup/ibex_gui
# Ticket2587 Convert Runcontrol Menu

### Description of work

Converted the run control menu to work in e4.

### To test

* Open the run control menu item
* Confirm that the panel functions as in e3
* Confirm that https://github.com/ISISComputingGroup/IBEX/issues/2661 is fixed

---

#### Code Review

    - Manual system tests for the functionality should be added if there are no automated tests
    - Manual system tests can be removed from the template if they are covered by suitable automated tests

### Functional Tests


### Final Steps
